docs/install/upgrade.md

```diff
@@ -43,4 +43,10 @@ So, copying `notebook` and `conf` directory should be enough.
 
    ```
    bin/zeppelin-daemon.sh start
-   ```
\ No newline at end of file
+   ```
+
+# Migration Guide
+
+## Upgrading From Zeppelin 0.6 to 0.7
+
+ - From 0.7, we don't use ZEPPELIN_JAVA_OPTS as default value of ZEPPELIN_INTP_JAVA_OPTS and also the same for ZEPPELIN_MEM/ZEPPELIN_INTP_MEM. If user want to configure the jvm opts of interpreter process, please set ZEPPELIN_INTP_JAVA_OPTS and ZEPPELIN_INTP_MEM explicitly.
```

Comment:
@zjffdu 
Currently this line is rendered  as like below because of ` "_" `
![screen shot 2016-07-26 at 10 37 39 am](https://cloud.githubusercontent.com/assets/10060731/17123280/63e5446a-531d-11e6-8eeb-07ff764e4852.png)

So how about attaching back quotation mark to back & forth of each variable names? I mean 
```

 - From 0.7, we don't use `ZEPPELIN_JAVA_OPTS` as default value of `ZEPPELIN_INTP_JAVA_OPTS` and also the same for `ZEPPELIN_MEM/ZEPPELIN_INTP_MEM`. If you want to configure the jvm opts of interpreter process, please set `ZEPPELIN_INTP_JAVA_OPTS` and `ZEPPELIN_INTP_MEM` explicitly.
```

Then it will be rendered like this 
![screen shot 2016-07-26 at 10 37 05 am](https://cloud.githubusercontent.com/assets/10060731/17123350/fbd6f598-531d-11e6-83ad-6119b0151637.png)